crates/viewer/re_view_spatial/src/visualizers/transform_axes_3d.rs

```diff
@@ -69,44 +71,70 @@ impl VisualizerSystem for TransformAxes3DVisualizer {
         );
 
         for data_result in query.iter_visible_data_results(Self::identifier()) {
-            let Some(transform_info) = transform_info_for_entity_or_report_error(
-                transforms,
-                &data_result.entity_path,
-                &mut output,
-            ) else {
-                continue;
-            };
-
-            // Determine which transforms to draw axes at.
-            // For pinhole cameras, we draw at the pinhole location only.
-            // For normal entities, we iterate over all instance poses.
-            let transforms_to_draw: smallvec::SmallVec<[glam::Affine3A; 1]> =
-                if let Some(pinhole_tree_root_info) =
-                    transforms.pinhole_tree_root_info(transform_info.tree_root())
+            // We first add the axes for the entity transforms, because we want them to be drawn below
```

Comment:
what do you mean by draw below, you mean before? We don't do any distance sorting on opaque objects yet so this _is_ probably the case, but I'm not sure we'll guarantee that in the future 🤔, well I do think if we change to front to back sorting for opaque we'll use a stable sort so if you end up with the exact same transform twice it should still work :) 🤞

Reply:
ahhh. yes makes sense